Win32Emu.Tools.PeAnalyzer.Wasm/wwwroot/win16-api-status.json

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
Sorry, this isn't what I was after. Can we use something similar to how we do it with win32 modules?